docs/guidance/model/organization_classifications.md

```diff
@@ -0,0 +1,86 @@
+
+There are therefore two recommended options for publishing organization classifications.
+1. For classifications that have become standardized, there are specific OCDS extension fields that should be used. At present, this only applies to organization scale, which should be disclosed using the [party scale extension](https://extensions.open-contracting.org/en/extensions/partyScale/master/). This extension adds a `scale` field to the `parties.details` block, to indicate the size or scale of an organization, in particular commercial enterprises or economic operators.
+2. For non-standardized options, such as classifying forms of organization ownership, publishers should use the [organization classification extension](https://github.com/open-contracting-extensions/ocds_organizationClassification_extension). This extension adds a [`classifications`](../schema/reference/#classification) array to the `parties.details` block to enable the categorization of organizations. Each `classification.id` field should contain a code from the particular scheme given in the `classification.scheme` field. Details about the particular organization characteristic that is being disclosed should be provided in the `classification.description` field. The given `classification.scheme` may be an existing scheme (drawn from the open [`itemClassificationScheme` codelist](../../../schema/codelists/#item-classification-scheme)), or a local scheme for a particular publisher. In both cases, we encourage publishers to provide details of all schemes and classification codes used in their [publication policy](../publish/publication_policy), to help users understand the data.
+
+As fields become standardized through the use of option 2, the information can be migrated to _also_ be published via specific fields as in option 1. Publishsers may continue to publish the information in the organization classification extension to preserve backwards compatibility in data sets.
+
+A third not-recommended example approach using local extensions is also given below, for situations where neither of the above two approaches apply to a specific use case.
+
+## Worked examples
+
+### Option 1: standardized options
+
+#### Organization scale
+
+In the example below, Moldova has disclosed information about the 'Solcoci Prim' organization using the [party scale extension](https://extensions.open-contracting.org/en/extensions/partyScale/master/). The scale is given as 'micro', from the [partyScale.csv codelist](https://extensions.open-contracting.org/en/extensions/partyScale/master/codelists/).
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/organization-classification/moldova_organization_scale.json
+   :jsonpointer: /releases/0/parties
```

Comment:
I think is better if we can show full releases and just expand the relevant sections, so the readers would have good examples of releases and packages across all the documentation (this is just my opinion).

Reply:
Yes, this would be consistent with the other examples and clearer, thanks.